content/docs/started/workstation/minikube-linux.md

```diff
@@ -301,4 +303,27 @@ Read the following before execution of the notebook.
         ```
         hostPath:
           path: /mnt/data
-        ```
\ No newline at end of file
+        ```
+
+## Hints and Tips
```

Comment:
The Kubeflow style is to use *sentence case* rather than *title case* for headings within the page. So, change this heading to:

```
## Hints and tips
```

Reply:
Thanks @sarahmaddox!

Will change.